jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/thread/QueuedThreadPoolTest.java

```diff
@@ -833,7 +833,8 @@ public void testDump() throws Exception
         assertThat(count(dump, " - STARTED"), is(3));
         assertThat(dump, containsString(",3<=3<=4,i=1,r=2,"));
         assertThat(dump, containsString("[ReservedThreadExecutor@"));
-        assertThat(count(dump, "+> 0: ReservedThread@"), is(1));
+        // Flaky
+        //assertThat(count(dump, "+> 0: ReservedThread@"), is(1));
```

Comment:
I would just delete rather than comment out.  This approach is always going to be flaky

Reply:
deleted. cf55c3aa71f